app/src/main/java/com/belatrixsf/connect/ui/event/detail/EventDetailActivity.java

```diff
@@ -45,16 +48,19 @@
 /**
  * Created by icerrate on 27/06/2016.
  */
-public class EventDetailActivity extends BelatrixConnectActivity implements EventDetailFragmentListener {
+public class EventDetailActivity extends BelatrixConnectActivity implements EventDetailFragmentListener, FragmentListener{
```

Comment:
I think BelatrixConnectActivity alread have the implementation for the FragmentListener